package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "vng-api-common",
-  "version": "2.1.0",
+  "version": "2.1.1",
```

Comment:
If you run `npm install` it should automatically update `package-lock.json` as well

Reply:
In this PR https://github.com/maykinmedia/commonground-api-common/pull/24 I remove `package*.json` files, so I think soon it would not be needed